goprocess/goprocess_linux_test.go

```diff
@@ -1,11 +1,14 @@
 package goprocess
```

Comment:
Ideally, we'd keep these tests for all platforms. How about keeping these as is (as far as possible) and adding a Linux-specific test with the `HOST_PROC` "hack" in addition?

Reply:
since gopsutil's Process is not interface, so need to real process to test. Now I've added a new hack to bring some tests back to all arch.